python/tskit/tables.py

```diff
@@ -1815,3 +1817,23 @@ def unpack_strings(packed, offset, encoding="utf8"):
     :rtype: list[str]
     """
     return [b.decode(encoding) for b in unpack_bytes(packed, offset)]
+
+
+def to_np_int32(int_array):
```

Comment:
maybe this should be a private method? is there a common use case for tskit-users?

Reply:
Good point. I can’t think of any reason to make this public. So I guess I should simply add an underscore to start of the function name?

Reply:
I've given up on using underscores for this a while back, it's mostly pointless unless it's **really** something that you don't want people to use. The operational definition of private here is 'not listed in public documentation'.

If we want to keep the namespace clean we can modify what's exported in the ``msprime/__init__.py``, at the moment I think we just take everything, but we could selectively export functions and classes.